### PR TITLE
#5 Generate model Item

### LINKED ISSUE #5: Generate model Item
Table schema:
- Type: String - Name
- Type: Text - Description
  
  Item has belongs_to relationship with model Wishlist.

### PR DESCRIPTION
**Pull request by nodzy**
Closes #5

### CODE REVIEW

**Comment by student-tomasz** [comment]
👍

**Comment by agalkiewicz** [comment]
:+1:

**Comment by shadre** [comment]
:+1:

File: app/models/item.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Item < ApplicationRecord
+  validates :name, presence: true, length: { maximum: 30 }
+  validates :description, length: { maximum: 200 }
```
Comment (gotar) [inline app/models/item.rb]: there is special requirement to limit size? why not just present true?

Comment (nodzy) [inline app/models/item.rb]: it's because of our views - we wanted to make sure it would look good, and optionally change limit later

**Comment by student-tomasz** [comment]
👍

File: app/models/item.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Item < ApplicationRecord
+  validates :name, presence: true
```
Comment (gotar) [inline app/models/item.rb:2]: Maybe good idea to have Uniqe item? Not sure how item will work and what purpose of it

Comment (nodzy) [inline app/models/item.rb:2]: In our case item shouldn't be unique

**Comment by gotar** [comment]
Ok merge it

**Pull request merged**